docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -889,15 +890,44 @@ testcafe chrome my-tests --sf
 
 ### --ts-config-path \<path\>
 
-Enables TestCafe to use a custom [TypeScript configuration file](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options) and specifies its location.
+Deprecated as of [TestCafe v.X.X.X] in favour of the --compiler-options parameter.
+
+*Related configuration file property*: [tsConfigPath](configuration-file.md#tsconfigpath).
+
+### --compiler-options \<options\>
+
+Modifies TypeScript compilation settings.
+
+To list multiple parameters, separate them with semicolons. Enclose values that contain spaces in quotes.
 
 ```sh
-testcafe chrome my-tests --ts-config-path /Users/s.johnson/testcafe/tsconfig.json
+testcafe chrome my-tests --compiler-options typescript.options.lib=ES5,WebWorker;typescript.typesRoot='this value contains spaces'
 ```
 
-You can specify an absolute or relative path. Relative paths are resolved against the current directory (the directory from which you run TestCafe).
+The `--ts-config-path` option is now deprecated. Specify the `typescript.configPath` compiler option to load an external TypeScript configuration file:
 
-*Related configuration file property*: [tsConfigPath](configuration-file.md#tsconfigpath).
+```sh
+testcafe chrome my-tests --compiler-options typescript.configPath = 'config file path'
+```
+
+Be careful when setting the value of the typescript.customCompilerModulePath compiler option. TestCafe resolves relative paths against its own installation folder. 
+
+Here’s how to enable the typescript 4 compiler included with your project instead of the typescript 3 compiler that ships with TestCafe.
```

Comment:
What about such a variant?
```js
Here's how to turn on the compilation tests using the `typescript@4` compiler instead of the default `typescript@3` one.
```

Reply:
Could you elaborate? What is it that you want to convey that, in your opinion, my sentence does not?

Reply:
You phrase is not strict and technically incorrect.

> Here’s how to enable the typescript 4 compiler included with your project
Enable for what? 
`compiler included with your project` - we can remove this phrase because compiler can be outside of project.
`Typescript 4 compiler` is bad term. Better to use the `typescript@4`.